src/components/Comments/WriteComment.js

```diff
@@ -68,7 +68,7 @@
     if (isKeyboardOpen) {
       commentInputRef.current.focus();
     }
-  }, [isKeyboardOpen, commentInputRef.current]);
+  }, [isKeyboardOpen, commentInputRef?.current]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The useEffect dependency array has been updated to include `commentInputRef?.current` instead of `commentInputRef.current`. This change is good because it ensures that the effect will re-run whenever the current value of the ref changes, not just when the ref object itself changes.
```diff
-  }, [isKeyboardOpen, commentInputRef.current]);
+  }, [isKeyboardOpen, commentInputRef?.current]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->